grouper/email_util.py

```diff
@@ -256,3 +258,47 @@ def notify_edge_expiration(settings, session, edge):
         settings=settings,
         context=email_context,
     )
+
+
+def notify_nonauditor_flagged(settings, session, edge):
+    """Send notification that a nonauditor in an audited group has had their membership
+    set to expire.
+
+    Handles email notification and audit logging.
+
+    Args:
+        settings (Settings): Grouper Settings object for current run.
+        session (Session): Object for db session.
+        edge (GroupEdge): The expiring edge.
+    """
+    # TODO(rra): Arbitrarily use the first listed owner of the group from which membership expired
```

Comment:
I made the user being flagged the actor. What is the right way you're envisioning? The only other solution I can think of would be to have a fake system level user be the actor, but I don't know if that's preferable to having the affected user be the actor, since the system level user could be non-obvious to someone who doesn't know of it's existence.